src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -21,7 +21,7 @@ internal static class PathUtilities
     {
         // We consider '/' a directory separator on Unix like systems. 
         // On Windows both / and \ are equally accepted.
-        internal static readonly char DirectorySeparatorChar = PlatformInformation.IsUnix ? '/' : '\\';
+        internal static char DirectorySeparatorChar => Path.DirectorySeparatorChar;
```

Comment:
why not keep a static-readonly?  once computed, it's not going ot change :)

Reply:
@CyrusNajmabadi This was suggested by Aleksey.

Reply:
ah, lol.  ok :)